docs/setup-development.md

```diff
@@ -154,3 +155,35 @@ npm run storybook
 ```
 
 It will start automatically on localhost:6006 and open a new tab in your browser.
+
+### Windows 11
+
+If you are running windows 11 your best option may be to use Windows Subsystem for Linux(WSL)
+
+#### Install WSL
+
+First you will need to make sure virtualization is enabled in your BIOS
+
+[Enable Virtualization](https://support.microsoft.com/en-us/windows/enable-virtualization-on-windows-11-pcs-c5578302-6e43-4b4b-a449-8ced115f58e1)
+
+Install WSL
+
+[WSL install](https://learn.microsoft.com/en-us/windows/wsl/install)
+
+You should now have an icon for "Ubuntu" your app drawer, alternatively you can open up the windows terminal, hit the down arrow and select Ubuntu.
+
+You will need to set your gitconfig email/username, and make a new folder to keep your projects in
+
+#### Install node
+
+[Install Node.js on WSL](https://learn.microsoft.com/en-us/windows/dev-environment/javascript/nodejs-on-wsl)
+
+#### Finish setup
+
+You should now be able to setup a repo as described in the Setup section at the top of this page.
```

Comment:
I'm not 100% how to do it but I think we can link to the header above directly here
```suggestion
You should now be able to setup a repo as described in the [Setup](./CONTRIBUTING.md#setup) section at the top of this page.
```